lib/url.js

```diff
@@ -533,8 +728,8 @@ Url.prototype.resolveObject = function(relative) {
   var mustEndAbs = (isRelAbs || isSourceAbs ||
                     (result.host && relative.pathname));
   const removeAllDots = mustEndAbs;
-  let srcPath = result.pathname && result.pathname.split('/') || [];
-  const relPath = relative.pathname && relative.pathname.split('/') || [];
+  var srcPath = result.pathname && result.pathname.split('/') || [];
+  var relPath = relative.pathname && relative.pathname.split('/') || [];
```

Comment:
The `resolveObject()` changes were mostly unintentional as the `url` module was modified while I was working on this. However, the reverting of `let` *is* intentional as `let` is still slow in v8. I've left the `const` as-is though.